examples/farmer/farmer_rho_demo.bash

```diff
@@ -2,7 +2,11 @@
 
 SOLVERNAME="cplex"
 
-mpiexec -np 3 python -m mpi4py farmer_rho_demo.py --num-scens 3 --bundles-per-rank=0 --max-iterations=1000 --default-rho=1 --solver-name=${SOLVERNAME} --xhatpath=./_farmer_cyl_nonants.npy --order-stat 0.5 --xhatshuffle --lagrangian --max-stalled-iters 5000 --rho-setter --rel-gap 0.001
+# get an xhat
+# xhat output file name is hardwired to 'farmer_cyl_nonants.npy'
+mpiexec -np 3 python -m mpi4py farmer_cylinders.py  --num-scens 3 --lagrangian --xhatshuffle --bundles-per-rank=0 --max-iterations=50 --default-rho=1 --solver-name=${SOLVERNAME}
+
+mpiexec -np 3 python -m mpi4py farmer_rho_demo.py --num-scens 3 --bundles-per-rank=0 --max-iterations=100 --default-rho=1 --solver-name=${SOLVERNAME} --xhatpath=./farmer_cyl_nonants.npy --grad-order-stat 0.5 --xhatshuffle --lagrangian --max-stalled-iters 5000 --grad-rho-setter --rel-gap 0.001
```

Comment:
Nit - but for consistency - shouldn't --xhatpath really be --xhatfile?

Reply:
He has both and I have been meaning to sort this out.